pages/claimstatus/index.tsx

```diff
@@ -18,16 +35,87 @@ export default function Home(): ReactElement {
         <title>{t('title')}</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-
       <Header />
       <Main />
       <Footer />
+      {console.log("-Test Response-")} {/* testing, to be removed before launch */}
+      {console.dir({claimData})} {/* testing */}
     </Container>
   )
 }
 
-export const getServerSideProps: GetServerSideProps = async ({ locale }) => ({
-  props: {
-    ...(await serverSideTranslations(locale || 'en', ['common', 'header', 'footer'])),
-  },
-})
+export const getServerSideProps: GetServerSideProps = async ({ req, locale }) => {
+  const PASSWORD = process.env.CERTIFICATE_PASSPHRASE
+  const API_URL: string = process.env.API_URL || "fail gracefully" //TODO: Fail gracefully. 
+  // const UNIQUE_NUMBER_HEADER: string = process.env.UNIQUE_NUMBER_HEADER_TITLE || "x-unique-number"
+  const CERT_DIR: (string) = (process.env.CERTIFICATE_DIR || "/var/ssl")
```

Comment:
should there be a `|| "var/ssl"` here?

Reply:
It's not the worst thing in the world, but I agree it should probably be removed.